Core/clim-core/presentations-clipboard.lisp

```diff
@@ -0,0 +1,24 @@
+(in-package :clim-internals)
+
+(define-presentation-generic-function %convert-clipboard-content convert-clipboard-content
+  (type-key parameters options object type output-type check-only))
+
+(define-presentation-method convert-clipboard-content (obj (type t) output-type check-only)
+  nil)
+
+(define-presentation-method convert-clipboard-content (obj (type string) (output-type (eql :string)) check-only)
+  (check-type obj string)
+  obj)
+
+(defun convert-clipboard-content (obj output-type &key type check-only)
+  (funcall-presentation-generic-function convert-clipboard-content
+                                         obj
+                                         (or type (presentation-type-of obj))
+                                         output-type
+                                         check-only))
+
+(defun supported-clipboard-types (obj type)
+  (loop
+    for output-type in '(:string :html)
```

Comment:
this shows string html, but representation-type-name has also image. how does it work in practice?

Reply:
This is intentional. Support for sending images to external processes has not been implemented.

Reply:
Oops. It's not intentional. I thinking of the wrong file. Fixed.